frontends/main/src/app-pages/LearningPathDetailsPage/ListDetailsPage.tsx

```diff
@@ -23,7 +23,7 @@ const ListDetailsPage: React.FC<ItemsListingComponentProps> = ({
       src="/images/backgrounds/course_search_banner.png"
       className="learningpaths-page"
     >
-      {/* TODO <MetaTags title={list?.title} social={false} /> */}
+      <MetaTags title={list?.title} />
```

Comment:
**Inference:** this route is using `<MetaTags />` and not the nextjs metadata api because some learning paths are private, and we don't do authentication on the nextjs server, and the metadata api is server-only, so we can't generate the metadata on the server. Is that right?

That said, I don't think this is doing anything. It looks like `MetaTags` uses `next/head`, which I believe is intended for the App router. (If you switch the `next/head` [docs](https://nextjs.org/docs/pages/api-reference/components/head) to the "App Router", the docs page disappears.)

**Suggestion / notes:** I think you should
- Replace `MetaTags.tsx` with something like `PrivateTitle` below, and use that here
- the title won't be server-rendered (though potentially we could for public learning paths, down the road) but will still show up client-side for things like tab titles.

```tsx
import React from "react"

/**
 * Alert! Prefer the nextjs [metadata api](https://nextjs.org/docs/app/building-your-application/optimizing/metadata)
 * This is intended for rendering titles that require authentication and hence
 * can't be rendered on our nextjs server.
 *
 * Render a title tag. Will be hoisted to <head> by React.
 * See [title](https://react.dev/reference/react-dom/components/title)
 */
const PrivateTitle: React.FC<{ title: string }> = ({ title }) => {
  return <title>{[title, process.env.NEXT_PUBLIC_SITE_NAME].join(" | ")}</title>
}

export default PrivateTitle
```

Note: If you look at the react docs for [title](https://react.dev/reference/react-dom/components/title), you'll see it says Canary. That should be ok. They are ["officially supported"](https://react.dev/blog/2023/05/03/react-canaries) and what Nextjs uses.

Reply:
Switched this to "privatetitle"